internal/backend/local/backend.go

```diff
@@ -106,9 +106,6 @@ func New(enc encryption.StateEncryption) *Local {
 // NewWithBackend returns a new local backend initialized with a
 // dedicated backend for non-enhanced behavior.
 func NewWithBackend(backend backend.Backend, enc encryption.StateEncryption) *Local {
-	if backend == nil && enc == nil {
```

Comment:
This is actually valid for some commands that do not use encryption, like get schema